docker/pact-stub/pact_stub.go

```diff
@@ -216,19 +216,17 @@ func getIgnoredInteractionsForCypressTesting(interactions []Interaction) []Inter
 	// We need this test to produce specific data in the response so that Cypress tests will pass.
 	// Since Pact won't let us return multiple array entries from `dsl.EachLike`
 	// we modify pact stub to return the desired output.
-	var updatedInteractions []Interaction
-	for _, i := range interactions {
-		switch i.ProviderState {
-		case "Some document template ids types exist":
-			i.Response.Body = []map[string]interface{}{{
+	for index, interaction := range interactions {
+		if interaction.Request.Path == "/lpa-api/v1/reference-data/documentTemplateId" {
+			interactions[index].Response.Body = []map[string]interface{}{{
```

Comment:
Happened across this bit of code and wanted to tidy it up a bit so it's not dependent on provider state text